wemake_python_styleguide/logic/tree/classes.py

```diff
@@ -34,3 +39,54 @@ def is_forbidden_super_class(class_name: Optional[str]) -> bool:
     if class_name in ALLOWED_BUILTIN_CLASSES:
         return False
     return is_builtin_name(class_name)
+
+
+def get_attributes(
+    node: ast.ClassDef,
+) -> Tuple[List[types.AnyAssign], List[ast.Attribute]]:
+    """Returns all class and instance attributes of a class."""
+    class_attributes = []
+    instance_attributes = []
+
+    for sub in ast.walk(node):
+        if isinstance(sub, ast.Attribute) and isinstance(sub.ctx, ast.Store):
+            instance_attributes.append(sub)
+            continue
+
+        has_assign = (
+            nodes.get_context(sub) == node and
+            getattr(sub, 'value', None)
+        )
+        if isinstance(sub, AssignNodes) and has_assign:
+            class_attributes.append(sub)
+
+    return class_attributes, instance_attributes
+
+
+def getter_setter_postfixes(node: ast.ClassDef) -> Set[str]:
+    """
+    Return postfixes of all getter or setter methods.
+
+    get_class_attribute becomes class_attribute
+
+    set_instance_attribute becomes instance_attribute
+
+    """
+    method_postfixes = set()
+    for subnode in ast.walk(node):
+        correct_context = nodes.get_context(subnode) == node
+        if isinstance(subnode, FunctionNodes) and correct_context:
+            if is_getter_or_setter(subnode):
+                method_postfixes.add(subnode.name.partition('get_')[2])
+                method_postfixes.add(subnode.name.partition('set_')[2])
+    return method_postfixes
+
+
+def is_getter_or_setter(node: types.AnyFunctionDef) -> bool:
+    """Checks if non property decorated function contains get or set prefix."""
+    if any(node.name.startswith(prefix) for prefix in ('get_', 'set_')):
+        is_property = functions.check_decorators(node, 'property')
```

Comment:
Since we are replacing `in` with `==` we need to generate a full name here. It is possible?

Reply:
"property" and ".setter" seems to be the full `name that node_to_string(decorator)` returns.

Reply:
But from earlier comments you wrote i was under the impression that decorators do not have to be checked anymore since the naming `get_x` and `set_x` should be forbidden no matter the decorator. So i was thinking about removing the decorator checking part.

Reply:
This is ok 👍

Reply:
So i should remove the decorator checking?

Reply:
Yes, please 👍